content/awards_content.php

```diff
@@ -182,7 +182,7 @@
 <div class="post">
     <div class="entry">
         <div class="captioned_image">
-            <img src="images/Ophy.JPG" />
+            <a href="http://www.thebluealliance.com/team/2471/2010"><img src="images/Ophy.JPG" /></a>
```

Comment:
This isn't blocking the merge, but I noticed there is no alt here which is inconsistent with every other robot.